pkg/install.go

```diff
@@ -197,55 +76,57 @@ func InstallGoChaosExperiment(testsDetails *types.TestDetails, experimentNamespa
 	return nil
 }
 
-//InstallGoChaosEngine installs the given ansible based chaos engine
+//InstallGoChaosEngine installs the given go based chaos engine
 func InstallGoChaosEngine(testsDetails *types.TestDetails, engineNamespace string) error {
 
 	// Fetch Chaos Engine
 	var out bytes.Buffer
 	var stderr bytes.Buffer
-	if err = DownloadFile(testsDetails.ExperimentName+"-ce.yaml", testsDetails.EnginePath); err != nil {
+	if err = DownloadFile("/tmp/"+testsDetails.ExperimentName+"-ce.yaml", testsDetails.EnginePath); err != nil {
```

Comment:
Any specific reason for /tmp?

Reply:
Just to avoid downloading too many files while running the test.
If we run two tests today it will download 6 new files in the test directory. This is to avoid this 
Related to issue - https://github.com/litmuschaos/litmus/issues/1532